avro2tf/src/main/scala/com/linkedin/avro2tf/jobs/TensorMetadataGeneration.scala

```diff
@@ -101,19 +100,21 @@ class TensorMetadataGeneration {
    * @return A mapping of column name to its cardinality
    */
   private def getColsOfIntOrLongCardinalityMapping(dataFrame: DataFrame, params: TensorizeInParams): Map[String, Long] = {
-
-    val colsContainIntOrLongType = TensorizeInConfigHelper.concatFeaturesAndLabels(params)
+    val intOrLongColNames = TensorizeInConfigHelper.concatFeaturesAndLabels(params)
       .map(featureOrLabel => featureOrLabel.outputTensorInfo.name)
       .filter(columnName => dataFrame.schema(columnName).dataType.isInstanceOf[IntegerType] ||
         dataFrame.schema(columnName).dataType.isInstanceOf[LongType])
 
+    val intOrLongCols = intOrLongColNames
```

Comment:
After the empty test is added here, we can check it the code. Thanks a lot for the contribution!